pkg/output/progress_test.go

```diff
@@ -7,13 +7,20 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestProgressTimeout(t *testing.T) {
+func TestProgressTimeoutDoesNotInc(t *testing.T) {
+	progress := NewProgress()
+	progress.Start()
+	progress.Stop() // should not hang
+	progress.Inc()  // should not hang
+	assert.Equal(t, progress.Val(), uint64(0))
+}
+
+func TestProgressTimeoutDoesNotHang(t *testing.T) {
 	progress := NewProgress()
 	progress.Start()
 	time.Sleep(progressTimeout + 1)
 	progress.Inc()  // should not hang
 	progress.Stop() // should not hang
-	assert.Equal(t, uint64(0), progress.Val())
```

Comment:
Isn't it relevant to assert this value?

Reply:
not realy but I changed the test to be sure the timeout was reached so I could put this test back